app/src/main/java/org/simple/clinic/summary/bloodpressures/newbpsummary/NewBloodPressureSummaryViewEffect.kt

```diff
@@ -10,3 +10,5 @@ data class LoadBloodPressures(val patientUuid: UUID) : NewBloodPressureSummaryVi
 data class OpenBloodPressureEntrySheet(val patientUuid: UUID) : NewBloodPressureSummaryViewEffect()
```

Comment:
Minor grammar correction in the commit message:

- "See all" button is **only shown** if there are more than 3 blood pressures